### DISCUSSION TITLE
Can't seem to configure dressing

### DISCUSSION THREAD

**Discussion by r-darwish** [LazyVim/LazyVim#482] [Q&A] [answered]
I wanted [configure dressing](https://github.com/stevearc/dressing.nvim#configuration) so that I'll have normal mode in `input`. I tried the straightforward approach:
```lua
{
  "stevearc/dressing.nvim",
  opts = { input = {
    insert_only = false,
  } },
},
```
But that didn't work. I noticed that LazyVim loads dressing [with a hook](https://github.com/LazyVim/LazyVim/blob/b48a1969d3f194ac96ffad437ca5d71a07545f44/lua/lazyvim/plugins/ui.lua#L45), so I tried two different methods:

```lua
{
  "stevearc/dressing.nvim",
  config = function()
    require("dressing.config").update({
      input = {
        insert_only = false,
      },
    })
  end,
}
```
and
```lua
{
  "stevearc/dressing.nvim",
  config = function()
    require("dressing").setup({
      input = {
        insert_only = false,
      },
    })
  end,
}
```

But none of these helped.

**Comment by folke** [comment, marked as answer]
Set `init = function() end` and `lazy = false` and then set `opts=....`

**Reply by r-darwish** [reply]
Thanks for the quick reply. That will didn't work but I'm beginning to wonder whether my question is wrong. I'm using nvim-dap-python which [calls the input function](https://github.com/mfussenegger/nvim-dap-python/blob/master/lua/dap-python.lua#L180-L183) to select argument. What I see is this pop-up:
![image](https://user-images.githubusercontent.com/1383193/227134309-920f8528-ea07-444a-a1e4-d2957751ef4b.png)
As I understand this is dressing's popup, but nothing I do seems to effect it. Even if I set `enabled = false` for input I still see it.

**Reply by folke** [reply]
If you still see it after disabling the plugin, then another plugin seems to be doing that

**Reply by r-darwish** [reply]
Yeah, I seem to have confused dressing with noice.
